Sources/SmileID/Classes/Util.swift

```diff
@@ -54,3 +54,65 @@ extension String {
         return self.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty ? nil : self
     }
 }
+
+func toErrorMessage(error: SmileIDError) -> (String, String?) {
+    switch error {
+    case .api(let code, let message):
+        let systemError = ["2201", "2301", "2401"]
+        let notAuthorized = ["2205", "2405"]
+        let missingParameters = ["2213", "2413"]
+        
+        if systemError.contains(code) {
+            return ("Si.Error.Message.2201.2301.2401", message)
+        } else if notAuthorized.contains(code) {
+            return ("Si.Error.Message.2205.2405", message)
+        } else if missingParameters.contains(code) {
+            return ("Si.Error.Message.2213.2413", message)
+        }
+        
+        switch code {
+        case "2203":
+            return ("Si.Error.Message.2203", message)
```

Comment:
We should be able to construct this dynamically, something like ("Si.Error.Message.\(code)", message)

Reply:
@jumaallan I see that we are still doing a mapping, instead of just constructing the resource string dynamically

Reply:
Pushed that fix